component/storage/indexeddb/indexeddb.go

```diff
@@ -143,6 +204,20 @@ func (s *store) Put(key string, value []byte, _ ...storage.Tag) error {
 	m["key"] = key
 	m["value"] = string(value)
 
+	if len(tags) > 0 {
```

Comment:
do these tags have to match the store's index configuration?

Reply:
For this "emulated" implementation, they do not. The store configuration in this PR is really just here to ensure consistent behaviour across store implementations (the common tests I've created mandate a certain minimum set of behaviours). When I do the real one, the tags will need to match the store's config.